### DISCUSSION TITLE
Export PyTorch model to ONNX model

### DISCUSSION THREAD

**Discussion by WillBao33** [facebookresearch/detectron2#3776] [Q&A]
I trained a Retinanet model using the existing config file from Detectron2 with the following command:
`python train.py --output_path ./retina --yaml COCO-Detection/retinanet_R_101_FPN_3x.yaml --resume --max_iter 20000 --lr_rate 1e-5 --batch_size 16 --iou 0.4`
I wonder how to use the caffe2_export.py to export the PyTorch file (.pth) to the ONNX file. What's the command that I should use and do I need to parse any arguments in the command? 
If this is not the right way to export the ONNX, could you suggest some methods to do it? 
Thanks

**Comment by thiagocrepaldi** [comment]
I am working on adding support to ONNX export (with and without caffe2) https://github.com/facebookresearch/detectron2/pull/4120 and https://github.com/facebookresearch/detectron2/pull/4153/

Hopefully it gets accepted soon
